isso/js/app/template.js

```diff
@@ -0,0 +1,93 @@
+const runtime = require("pug-runtime");
+const utils = require("app/utils");
+
+// TODO get rid of pug-loader entirely
```

Comment:
I'm not familiar with `pug`:
is the work in this file finished, or are there things that remain to be properly configured / cleaned up?

Reply:
I should have clarified this: `pug-loader` allows one simplify loading the templates, loading them as "modules" right inside the `require` module loading syntax.

So one can do `require("pug!path/to/templates/xyz.pug")` instead of figuring out how to load templates from disk. This "loader" needs to be configured in `webpack`, as I have done.

However, using `pug-loader` pulls in _another_ dependency, and, most importantly, clashes with `Jest` testing (Jest is not able to recognize this loader syntax).

Will try to amend commit with this info.

Reply:
Amended with comment.